src/test/java/edu/uci/ics/cs221/index/positional/Team2CompressorTest.java

```diff
@@ -0,0 +1,102 @@
+package edu.uci.ics.cs221.index.positional;
+
+import edu.uci.ics.cs221.index.inverted.DeltaVarLenCompressor;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+
+public class Team2CompressorTest {
+
+    private DeltaVarLenCompressor compressor;
+
+    @Before
+    public void init() {
+        compressor = new DeltaVarLenCompressor();
+        assertNotNull(compressor);
+    }
+
+    public void fullTest(List<Integer> ints, byte[] bytes) {
+        assertEquals(compressor.encode(ints), bytes); // To test encode function
```

Comment:
```suggestion
        assertArrayEquals(bytes, compressor.encode(ints)); // To test encode function
```
and add relevant import:
```
import static org.junit.Assert.assertArrayEquals;
```